libs/platform/user/ebpf_platform_user.cpp

```diff
@@ -901,8 +901,6 @@ _ebpf_preemptible_routine(_Inout_ PTP_CALLBACK_INSTANCE instance, _In_opt_ void*
 
     ebpf_preemptible_work_item_t* work_item = (ebpf_preemptible_work_item_t*)parameter;
     work_item->work_item_routine(work_item->work_item_context);
-
-    ebpf_free_preemptible_work_item(work_item);
```

Comment:
please revert.

Reply:
Sure, still needs investigation as it crashes ("Reason: Value unavailable error for BadAddress")here when added

`7e9ff5f0 00007ff6`a555e180     unit_tests!ebpf_free+0x48 [C:\ebpf\ebpf-for-windows\libs\platform\user\ebpf_platform_user.cpp @ 449] 
09 000000f9`7e9ff630 00007ff6`a555bcae     unit_tests!ebpf_free_preemptible_work_item+0x30 [C:\ebpf\ebpf-for-windows\libs\platform\user\ebpf_platform_user.cpp @ 917] 
0a 000000f9`7e9ff660 00007ff9`d6d5236a     unit_tests!_ebpf_preemptible_routine+0x5e [C:\ebpf\ebpf-for-windows\libs\platform\user\ebpf_platform_user.cpp @ 906]